spec/views_test/user_index_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../rails_helper'
+
+RSpec.describe 'user_index', type: :feature do
+  before(:each) do
+    DatabaseCleaner.clean
+    @user = User.create(
+      name: 'zubair',
+      photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+      bio: 'full-stack developer.',
+      posts_counter: 0
+    )
+    visit root_path
+  end
+
+  it 'shows the username of the users' do
+    expect(page).to have_content("#{@user.name}")
+  end
+
+  it 'shows the photo of the user' do
+    expect(page.html).to include('user-img')
+  end
+
+  it 'shows the number of posts' do
+    expect(page).to have_content("#{@user.posts_counter}")
+  end
+
+  it 'should redirect to user show page when a name is clicked' do
+    user = User.first
+    click_link(user.name)
+    expect(page.current_path).to eql("/users/#{user.id}")
+  end
```

Comment:
- [ ] You did a nice job but make sure all your test cases are working
<img width="832" alt="image" src="https://user-images.githubusercontent.com/37901469/199478582-9b5c741b-a5f8-4958-a529-2bee0ddfde22.png">

Reply:
hi @m05e5  
its because of you test dataset it may contain different records 
the test is working for me as you can see 
![test](https://user-images.githubusercontent.com/95174521/199486130-f0ffff32-3300-4a40-94fc-f2edd02248f7.png)

Reply:
i fixed it here is a prove 
![test](https://user-images.githubusercontent.com/95174521/199488346-64043ddf-1eaa-46c3-88bf-4fd213294a84.png)